crates/red_knot_project/src/db/changes.rs

```diff
@@ -144,6 +145,12 @@ impl ProjectDatabase {
                         metadata.apply_cli_options(cli_options.clone());
                     }
 
+                    if let Err(error) = metadata.apply_configuration_files(self.system()) {
+                        tracing::error!(
+                            "Failed to apply configuration files, continuing without applying them: {error}"
+                        );
+                    }
```

Comment:
We may want to use proper diagnostics here so that LSP users get notified about this error but there are other errors in this file where we're using `tracing` -> A problem for another day